pkgs/development/tools/database/prisma-engines/default.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub
+, lib
+, openssl
+, pkg-config
+, protobuf
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "prisma-engines";
+  version = "2.30.0";
```

Comment:
2.30.2 is the latest. You might want to init on that. 3.0.1 is out on next Tuesday.